pkgs/by-name/cl/claude-squad/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  tmux,
+  gh,
+}:
+
+buildGoModule rec {
+  pname = "claude-squad";
+  version = "1.0.8";
+
+  src = fetchFromGitHub {
+    owner = "smtg-ai";
+    repo = "claude-squad";
+    rev = "v${version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```
Most of forge fetchers now use `tag` for fetching on tags